android/modules/ui/src/java/ti/modules/titanium/ui/widget/tableview/TableViewHolder.java

```diff
@@ -209,13 +225,15 @@ public void bind(final TableViewRowProxy proxy, final boolean selected)
 			// Handle row left and right images.
 			if (properties.containsKeyAndNotNull(TiC.PROPERTY_LEFT_IMAGE)) {
 				final String url = properties.getString(TiC.PROPERTY_LEFT_IMAGE);
-				final Drawable drawable = fileHelper.loadDrawable(url, false);
+				final TiDrawableReference drawableReference = TiDrawableReference.fromUrl(proxy, url);
+				final Drawable drawable = drawableReference.getDrawable();
 				this.leftImage.setImageDrawable(drawable);
 				this.leftImage.setVisibility(View.VISIBLE);
 			}
 			if (properties.containsKeyAndNotNull(TiC.PROPERTY_RIGHT_IMAGE)) {
 				final String url = properties.getString(TiC.PROPERTY_RIGHT_IMAGE);
-				final Drawable drawable = fileHelper.loadDrawable(url, false);
+				final TiDrawableReference drawableReference = TiDrawableReference.fromUrl(proxy, url);
+				final Drawable drawable = drawableReference.getDrawable();
```

Comment:
This is loading images in APK "assets" as-is instead of density scaling them. Which means images are appearing smaller than they were before.

I think a better solution is to use the `TiUIHelper.getResourceDrawable()` method instead. This correctly fetches from the "res" folder and also supports scaling asset images. We use it for `TabGroup` and `TabbedBar`.